File: reverseSentence/reverseSentence.js
function reverseWords(str) {
  return str
    .split(' ')
    .map((str) => str.split('').reverse().join(''))
    .join(' ');
}

//js good soln:
const reversedWords = (str) =>
  str.split('').reverse().join('').split(' ').reverse().join(' ');
// or

console.log(reverseWords('The quick brown fox jumps over the lazy dog.'));

File: reverseSentence/reverseSentence.py
def reverse_words(text):
    reversed = text.split(" ")
    arr = []
    for i in reversed:
        arr.append(i[::-1])
    joined = " ".join(arr)
    return joined

# best soln: comprehension


def reverse_words(str):
    return ' '.join(s[::-1] for s in str.split(' '))
